common/app/model/pressedContent.scala

```diff
@@ -160,7 +220,7 @@ final case class PressedProperties(
   showKickerTag: Boolean,
   showByline: Boolean,
   imageSlideshowReplace: Boolean,
-  maybeContent: Option[ContentType],
+  maybeContent: Option[PressedStory],
```

Comment:
Really nice. I think the benefits of splitting the types at this point are numerous.